core/math/src/main/java/org/arakhne/afc/math/geometry/d2/d/MultiShape2d.java

```diff
@@ -222,6 +222,24 @@ public void shapeGeometryChange(Shape2d<?> shape) {
 			onContentGeometryChange();
 		}
 
+		public boolean equals(Object obj) {
+			if (!super.equals(obj)) {
```

Comment:
Should be:
```java
public boolean equals(Object obj) {
    return this.delegate.equals(obj);
}
```

Indeed, this class is a delegation design pattern and does no have any attributes except the delegate.